kolibri/core/assets/src/core-actions.js

```diff
@@ -170,6 +246,24 @@ function initContentSession(store, Kolibri, channelId, contentId, contentKind) {
       /* If a summary model exists, map that to the state */
       if (summary.length > 0) {
         store.dispatch('SET_LOGGING_SUMMARY_STATE', _contentSummaryLoggingState(summary[0]));
+        if (summary[0].currentmasterylog) {
```

Comment:
I don't quite understand what's special about the `0`'th summary. Are they ordered reverse-chronologically, making this the "current" (most recent) one?